jetty-websocket/websocket-core/src/test/java/org/eclipse/jetty/websocket/core/autobahn/AutobahnWebSocketNegotiator.java

```diff
@@ -85,6 +85,8 @@ public FrameHandler negotiate(Negotiation negotiation) throws IOException
     public void customize(FrameHandler.CoreSession session)
     {
         session.setIdleTimeout(Duration.ofMillis(5000));
```

Comment:
Ah here is the 5s!  Try setting this to 10s to see if that is the cause of some failures?